docs/registry/service-binding-registry/README.adoc

```diff
@@ -0,0 +1,597 @@
++
+The RHOAS Operator also creates a `KafkaConnection` or `ServiceRegistryConnection` object for your Kafka or {registry} instance, which connects the instance to the OpenShift cluster. When you bind your Kafka or {registry} instance to an application on OpenShift, the Service Binding Operator uses the `KafkaConnection` or `ServiceRegistryConnection` object to provide the application with the necessary connection information for the instance. Binding an application to your Kafka or {registry} instance is described later in this guide.
+
+. Enable the new service account created by the RHOAS Operator to access the Kafka or {registry} instance that you specified.
+.. If you connected to a Kafka instance, set Access Control List (ACL) permissions to enable the new service account to access resources in the Kafka instance.
++
+.Setting Kafka access permissions for the service account
+[source,options="nowrap",subs="+quotes"]
+----
+$ rhoas kafka acl grant-access --consumer --producer --service-account __<my-client-id>__ --topic "\*" --group "*"
+----
++
+In the preceding command, replace __<my-client-id>__ with the `Client ID` value for the new service account. This value was previously shown on the command line.
++
+You should see output like the following example:
++
+.Example output when setting Kafka access permissions
+[source,options="nowrap"]
+----
+The following ACL rules are to be created:
+
+  PRINCIPAL (7)                                   PERMISSION         DESCRIPTION
+ ----------------------------------------------- ------------------ ---------------------------
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | DESCRIBE   TOPIC is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | READ       TOPIC is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | READ       GROUP is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | WRITE      TOPIC is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | CREATE     TOPIC is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | WRITE      TRANSACTIONAL_ID is "*"
+  srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1   ALLOW | DESCRIBE   TRANSACTIONAL_ID is "*"
+
+? Are you sure you want to create the listed ACL rules (y/N) Yes
+✔️ ACLs successfully created in the Kafka instance "my-kafka-instance"
+----
++
+In this example, the permissions you create allow applications to use the service account to create and delete topics in the Kafka instance, to produce and consume messages in any topic in the instance, and to use any consumer group and any producer.
+
+.. If you connected to a {registry} instance, use Role-Based Access Control (RBAC) to enable the new service account to access the {registry} instance and the artifacts (such as schemas) that it contains.
```

Comment:
@EricWittmann From here down to line 401 are instructions about granting the service account created by the RHOAS Operator access to a service registry instance. Can you take a look?

Reply:
Looks goood